exercises/01_trim_right/01_trim_right.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+)
+
+const (
+	trimSet = " \r\n\t"
+)
+
+func InSet(b byte, set string) bool {
+	for _, v := range set {
+		if v == rune(b) {
+			return true
+		}
+	}
+	return false
+}
+
+func TrimRight(str string) string {
+	for i := len(str) - 1; i > 0 && InSet(str[i-1], trimSet); i-- {
+		str = str[:i-1]
+	}
+	return str
+}
+
+func main() {
+
+	fmt.Print("Enter a string: ")
```

Comment:
nit: It looks a bit strange in the command-line:
```cli
Enter a string: Enter text: 
```
I suggest either adding a string to explain what this program does, or just to drop this line.